ods_ci/run_interop.sh

```diff
@@ -65,14 +85,13 @@ if [ $retVal -ne 0 ]; then
     exit "$retVal"
 fi
 
-if [[ -z "${TEST_SUITE}" ]]; then
-  echo "Define TEST_SUITE"
-  exit 1
-fi
+if [[ ${TEST_SUITE} == "Pre-Upgrade" ]]; then
+  echo "Save test config file..."
+  cp test-variables.yml ${SHARED_DIR}/test-variables.yml
 
-if [[ -z "${ARTIFACT_DIR}" ]]; then
-  echo "Define ARTIFACT_DIR"
-  ARTIFACT_DIR="/tmp"
+  echo "Running pre-upgrade testing"
+  TEST_CASE_FILE="tests/Tests/0200__rhoai_upgrade/0201__pre_upgrade.robot"
```

Comment:
Since we changed the path of these files in 2.12 if you run it with 2.8 you need to use a different path like:
tests/Tests/100__deploy/120__upgrades/120__pre_upgrade.robot
So it will be better to include the Tag `PreUpgrade` which is good for all branches.